tests/unit/inspect/test_inspectdatasetmanager.py

```diff
@@ -829,3 +850,45 @@ def test_get_concepts_hierarchy_info_hierarchy_with_depth_more_than_one():
     assert isinstance(result.tree, Tree)
     assert result.tree.depth() == 2
     assert len(result.tree.all_nodes_itr()) == 10
+
+
+def test_melt_works_when_column_titled_value_exists():
+    """
+    When a df with a column titled "Value" is passed to the melt function in
+    pandas 2.0 a ValueError is produced because of the column title. This test
+    uses a dataset with a "Value" titled column and should be melted when
+    transform_dataset_to_canonical_shape is called. In this case, the "value_name
```

Comment:
Liking the detail on the docstring!

Reply:
Wondered if it was a bit long winded but I think it will give some context for the follow-up ticket #860